Resources/Maps/northway.yml

```
@@ -6757,6 +6740,11 @@
     - type: Transform
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)